README.md

```diff
@@ -1,6 +1,70 @@
-# noredink-ui
-
-UI widgets we use.
+# Ownership, policies, & key concepts
+
+NoRedInk’s accessibility team, the Accessibilibats, own `noredink-ui`/the shared component library.
+While others may contribute to the component library and are encouraged to do so, the Accessibilibats (a.k.a A11ybats) are responsible for oversight of the foundational aspects of the component library, a.k.a. “**[Component Library Foundations](#component-library-foundations)**.”
+
+Given this ownership and responsibility, A11ybats will provide guidance and support to developers and designers who are building new components or working with existing components.
+
+[The shared component library app can be found here](https://noredink-ui.netlify.app/).
+
+## Component Library Foundations
+- Adherence to the component library [accessibility policy](#accessibility-policy)
+- Adherence to [UI Style Guide (and Caveats)](https://paper.dropbox.com/doc/UI-Style-Guide-and-Caveats-PvOLxeX3oyujYEzdJx5pu) and [Guidelines for Sharing User-Facing Changes with Design](https://paper.dropbox.com/doc/Guidelines-for-Sharing-User-Facing-Changes-with-Design-bdKGQtYH9qO9I00hUkA6k) 
+- Interoperability and consistency of components with each other and with the NoRedInk app
```

Comment:
Not sure if this makes sense or is the right way to word it. Feel free to adjust or suggest changes. Also not sure if "consistency" sounds like we're responsible for design?

Reply:
It doesn't sound like we're responsible for design to me -- I think it's fine!